config/prow/plugins.yaml

```diff
@@ -537,11 +537,14 @@ retitle:
 
 cherry_pick_unapproved:
   comment: |
-    This cherry pick PR is for a release branch and has not yet been approved by the Patch Release Team.
+    This cherry pick PR is for a release branch and has not yet been approved by [Release Managers](https://git.k8s.io/sig-release/release-managers.md).
     Adding the `do-not-merge/cherry-pick-not-approved` label.
```

Comment:
Just out of curiosity: do you think we can switch over to the usual `/lgtm` and `/approve` workflow without having a need for this additional label in the future?

Reply:
@saschagrunert -- We still need patch review from a Release Manager to vet that a PR is actually appropriate for cherry pick.

Reply:
Okay thank you for the clarification. :)